apps/src/applab/applab.js

```diff
@@ -1879,6 +1888,27 @@ Applab.onDesignModeClear = function() {
   document.getElementById('divApplab').innerHTML = Applab.levelHtml = "";
 };
 
+Applab.onDesignModeManageAssets = function(assetChosen, typeFilter) {
+  var codeDiv = document.createElement('div');
+  var showChoseImageButton = assetChosen && typeof assetChosen === 'function';
+  var dialog = studioApp.createModalDialog({
+    Dialog: Dialog,
```

Comment:
I dug into this a little bit and I think this cleanup fits better as a separate PR.  Most of the code uses `appOptions.Dialog`, but a few places use the global `window.Dialog`.  It's weird to pass in the impl each time something wants to show a dialog, so I think you're right that we should move everything to `StudioApp` and always use `appOptions.Dialog` inside the function.

There's a `StubDialog` in `app.html.ejs` that is the fallback impl for the `grunt dev` sever (no Dashboard).